lib/location/Placemark.ts

```diff
@@ -44,4 +44,18 @@ export const placemarkToFormattedAddress = (placemark: Placemark) => {
   return formattedAddress === US_COUNTRY_CODE ? undefined : formattedAddress
 }
 
+/**
+ * Formats a `Placemark` instance into a short address (name, city, region/state)
+ *
+ * At the moment, this only outputs US style addresses.
+ */
+export const placemarkToAbbreviatedAddress = (placemark: Placemark) => {
+  if (placemark.city === undefined || placemark.region === undefined) {
+    return "Unknown Location"
+  } else if (placemark.name === undefined) {
+    return `${placemark.city}, ${placemark.region}`
```

Comment:
You can use `!placemark.xxx` to check if something is undefined in a more concise way. Additionally, I think this function is only used for the event card so you can definitely just move it in the same file.